hr_payroll_commission/models/hr_contract.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+# © 2016 Coninckx David (Open Net Sarl)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+
+from openerp import models, fields, api
+import openerp.addons.decimal_precision as dp
+
+
+class HrContract(models.Model):
+    _inherit = 'hr.contract'
+
+    # ---------- Fields management
+    commission = fields.Float(string='Commission',
```

Comment:
This name is misleading. is is not the amount of commissions, but rather the amount of invoiced sales eligible for commission.